README.md

```diff
@@ -1,2 +1,27 @@
-# workloads
-Collection of snapshots of WebAssembly workloads
+# WebAssembly Workloads
+
+This repo contains a collection of snapshots of WebAssembly workloads
+served at [webassembly.github.io/workloads](https://webassembly.github.io/workloads).
+
+The goals of the repo are:
+* to provide a common place to share workloads between developers of
+  WebAssembly applications, engines and toolchain
+* to provide meaningful measurements of realistic paths through realistic
+  codebases (i.e., not collect synthetic benchmarks)
+* to contain builds that can both be run locally and served via
+  GitHub Pages
+* to stay up-to-date with the latest recommended best practices and toolchain
+  developments (archiving snapshots of older builds);
+* to include sources whenever possible so that workloads can be rebuilt in order
+  to test toolchains and keep builds up-to-date
+
+## Contributing
+
+Do you have a workload that meets the goals above? We'd love to share it! Please
+feel free to create a PR against this repo.
+
+The workload should contain a LICENSE in the root directory that is sufficiently
+permissive as to allow:
+* copying the workload into various engine or toolchain test harnesses
+* hosting the workload via this and other GitHub Pages
+* modification of the workload for measurement and experimental purposes
```

Comment:
Build instructions.

Reply:
Good point; need to tweak a bit since this is just a bulleted list of license requirements.